README.md

```diff
@@ -27,24 +27,24 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 * How to configure zip deployment:
   * If you set the deployment source of your web app to “None” (the default source on app creation), the deploy command will Zip the contents of a selected folder and upload the Zip file to Azure.
 
-  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click "Yes", the following changes will be made in your project:
-    * The vscode setting 'appService.zipIgnorePattern' is changed to include build artifacts to be ignored for zipping.  These will be built on the server by running the appropriate build command.
-    * A '.deployment' file will be created in the root of the project.  This file configures SCM_DO_BUILD_DURING_DEPLOYMENT=true which enables build on deploy.
+  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click 'Yes', the following changes will be made in your project:
+    * The vscode setting `appService.zipIgnorePattern` is changed to indicate that build artifacts will not be zipped and deployed.  These will be built on the server by running the appropriate build command.
+    * A '.deployment' file will be created in the root of the project.  This file configures `SCM_DO_BUILD_DURING_DEPLOYMENT=true` enabling build on deploy.
       > NOTE: Currently only 'node' runtimes support this feature.
-  * If you select 'Never show again,' the 'appService.showBuildDuringDeployPrompt' vscode setting will be set to 'false' and you will no longer be prompted for this project.  Delete this setting or set it to 'true' to re-enable the prompt.
+  * If you select 'Never show again,' the `appService.showBuildDuringDeployPrompt` vscode setting will be set to `false` and you will no longer be prompted for this project.  Delete this setting or set it to `true` to re-enable the prompt.
 
   ### Zipdeploy Configuration Settings
 
-  * 'appService.zipGlobPattern'
-    * Uses a glob pattern to define which files to be included in the deployment. The default value is “**/*”.
+  * `appService.zipGlobPattern`
+    * Uses a glob pattern to define which files to be included in the deployment. The default value is '**/*'.
 
-  * 'appService.zipIgnorePattern'
+  * `appService.zipIgnorePattern`
     * Uses a glob pattern to define which files to be excluded from the deployment. The default value is [] which doesn’t exclude any files/folders.
 
-  * For example, you might want to exclude the “node_modules” folder from the deployment to speed up the Zip file creation and uploading. In this case, you will need the following setting:
-    * “appService.zipIgnorePattern”: ["node_modules{,/**}"]
+  * For example, you might want to exclude the 'node_modules' folder from the deployment to speed up the Zip file creation and uploading. In this case, you will need the following setting:
+    * `appService.zipIgnorePattern: ['node_modules{,/**}']`
```

Comment:
This should be `"appService.zipIgnorePattern": ["node_modules{,/**}"]`
If I copy your existing content into my settings.json, I get this:
![screen shot 2018-03-08 at 9 42 12 am](https://user-images.githubusercontent.com/11282622/37166822-00d3dd0a-22b5-11e8-90f2-1795e59e7d84.png)

It doesn't like single quotes